docs/quarantine.md

```diff
@@ -0,0 +1,63 @@
+# Test Quarantine
+
+The execution of tests on CI should be as deterministic as possible, besides
+potential issues coming from infrastructure or external dependencies. When a
+test sometimes fails and sometimes passes, apparently randomly, without any
+relationship with the changes in a PR, it has very negative implications:
+* The results of the whole suite are less valuable, we can no longer trust in
+the information given by the suite execution.
+* We can't be completely sure about the state of the product, it can be ok or
+not.
+* The development process itself is affected negatively, developers need to wait
+until a lucky execution aligns all the required tests in green, which can take
+more than a week in some cases.
+
+In summary, the CI system stops helping the product evolution and instead becomes
+an obstacle.
+
+We have introduced a quarantine methodology to put apart the tests that don't
+behave deterministically until they are fixed, so that we can keep the rest of
+the suite as healthy as possible.
+
+## Procedure
+
+In order to remove as much as possible the influence of changes in PRs to
+determine the stability of the suite, we will take into account only results
+from the periodics that run e2e tests from master, these jobs can be checked on
+testgrid [here](https://testgrid.k8s.io/kubevirt-periodics).
+
+In the following definitions we will consider as failures errors as reported on
+testgrid in jobs where less than 5 tests failed, so that we don't take into
+account systemic failures caused for instance by an infrastructure problem.
+
+### Putting tests in quarantine
+
+A test must be put in quarantine when:
+* It has failed at least 2 times in the last two weeks.
+* On each failed build less than 5 tests failed in total (to prevent the
+influence of systemic failures, like infrastructure problems).
+
+A PR will be proposed on mondays every two weeks with a batch of the tests that
+should be put in quarantine according to the data of the last two weeks. The PR
+will add the text `[QUARANTINE]` to each test's description in the code.
+
+Each quarantined test must have an team owner. The PR will add the text
+`[owner:@sig-{compute,network,storage}]` to each test's tdescription. Also, a
+tracking ticket will be created in [this board](https://issues.redhat.com/secure/RapidBoard.jspa?rapidView=8960)
```

Comment:
We cannot include a downstream resources like this in an upstream doc. You can still use it if you please, but it cannot be part of a formal KubeVirt procedure. How about using https://github.com/features/project-management/ ?

Reply:
also, for the ticket creation, not all tests will have test_id's associated with them. just pointing out that this isn't a reliable identifier

Reply:
Makes total sense to me, I've created this project https://github.com/kubevirt/kubevirt/projects/2?add_cards_query=is%3Aopen and linked it from the doc, let me know wdyt.

Reply:
In order to prevent duplication of efforts I've removed references to tickets and boards, PTAL.